sdk/storage/inc/datalake/file_system_client.hpp

```diff
@@ -117,6 +109,22 @@ namespace Azure { namespace Storage { namespace DataLake {
         const std::map<std::string, std::string>& metadata,
         const FileSystemSetMetadataOptions& options = FileSystemSetMetadataOptions()) const;
 
+    /**
+     * @brief Gets the metadata of file system.
+     * @param options Optional parameters to get the metadata of this file system.
+     * @return std::map<std::string, std::string>
+     */
+    std::map<std::string, std::string> GetMetadata(
```

Comment:
We don't need this API

Reply:
`GetProperties` covers metadata.

Reply:
However, `GetProperties` returns more than Metadata, that customer might not need. I don't see the harm in adding this API.

Reply:
No other SDK is doing so. Plus I do not think we can add API at will

Reply:
I can delete this if we are rejected in Jeffrey's review.

Reply:
Yes, GetProperties should be good enough